zulipterminal/model.py

```diff
@@ -126,7 +126,10 @@ def __init__(self, controller: Any) -> None:
 
         self.unread_counts = classify_unread_counts(self)
 
-        self.emoji_data = emoji_data.EMOJI_DATA
+        custom_emojis = self.fetch_custom_emojis()
+        all_emojis = (list(emoji_data.EMOJI_DATA.items())
+                      + list(custom_emojis.items()))
```

Comment:
Can we end up with duplicates here? Is that OK? Custom emoji can override unicode ones, I believe, but I'm not sure how we want to handle that situation.

Reply:
I didn't consider this. The dictionary has emoji name as keys so it cannot have duplicates.
Since custom emojis are added later, the unicode ones with the same names are quietly removed. This does seem to achieve what we want - the overriding is taken care of automatically - but is it okay to achieve it this way?